Extensions/AboutUsParser.js

```diff
@@ -0,0 +1,267 @@
+            <div>
+                <div class="card-title">
+                    <h3>${this.title}</h3>
+                </div>
+                <div class="card-text short">
+                    <p>${this.short}</p>
+                </div>
+            </div>
+            <div class="card-body">
+                <div class="card-text long">
+                ${this.long.map(photo =>`<p>${photo}</p>`).join('<br>')}
+                </div>
+            </div>
+            <div class="card-footer">
+                <button class="toggle-btn">...see more</button>
+            </div>
+        </div>
+    </div>`.trim();
+    }
+}
+
+function Files(path = null, result) {
+    path = path ? path : C.dPath;
+
+    getFiles(Path.join(path), (err, res) => {
+        if (err) throw new Error(err);
+        result(res.filter((file) => !fs.statSync(Path.join(file)).isDirectory()));
+    });
+}
+
+/**
+ * 
+ * @param {Array} files Array of files returned by function `Files(path, result)`
+ */
+async function createFilesArray(files) {
+    let mappedFiles = [];
+    files.map(file => {
+        file = file.split('/');
+        mappedFiles.push({ name: file[file.length - 1], path: Path.join(...file), language: file[file.length - 2], folder: file[file.length - 3] });
+    });
+    return mappedFiles;
+}
+
+/** Parses given string by separators and empty lines
+ * 
+ * @param {String} string text to be parsed
+ * @param {Function} data
+ * @returns {Array} 
+ */
+function parseString(string) {
+    try {
+        let sections = string.split(C.separator); // We split the string by our main separator into individual sections
+        let parsedSections = [];
+
+        sections.map(section => { // We iterate through every parsed section
+            let para = section.split('\r\n').filter(sentence => sentence.trim() ? true : false); // Splitting sections into paragraphs by new line and removing empty lines
+            //console.log(para.slice(2, para.length));
+            parsedSections.push({ title: para[0], short: para[1], long: para.slice(2,para.length) }); // Pushing a parsed section object into an array
+        });
+
+        return parsedSections; // return array of parsed sections as a callback
+    } catch (error) {
+        console.error(error);
+    }
+}
+
+/**
+ * 
+ * @param {Array} array 
+ */
+function getStringFromArray(array) {
+    let str_builder = '';
+    //console.log(array);
+    for (var par of array)
+        str_builder += new Paragraph(par).getDefaultHtmlCode();
+    
+    return str_builder;
+}
+
+/**
+ * @param {String} Fpath Path where to create the component
+ * @param {String} name File name of the component
+ * @param {String} string String that will be written inside the component
+ * @param {String} language
+ * @param {Boolean} append_file Whether to append or overwrite already existing file 
+ */
+async function createComponent(Fpath = null, name, string, language, append_file = true, fileName = 'school') {
```

Comment:
please let's not use abbreviations where not needed. this is so much harder to read

Reply:
and why is it with a capital?